client/app/components/visualizations/editor/withControlLabel.jsx

```diff
@@ -70,7 +71,12 @@ export default function withControlLabel(WrappedControl) {
 
     return (
       <ControlLabel layout={layout} label={label} labelProps={labelProps} disabled={disabled}>
-        <WrappedControl id={labelProps.htmlFor} disabled={disabled} {...props} />
+        <WrappedControl
+          className={cx("visualization-editor-input", className)}
```

Comment:
I forced all inputs to have `width: 100%` to remove the `w-100` classes (all the InputNumbers and Select's had this class). LMK if you see any isolated case as I didn't find any on my checks.

Reply:
There should be no special cases, as far as I remember all editor inputs should have `width: 100%` and we use grid where needed